setup.py

```diff
@@ -73,7 +76,7 @@
 extra_deps = {}
 
 extra_deps['dev'] = [
-    'pre-commit>=2.18.1,<3',
+    'pre-commit>=3.2.0,<3',
```

Comment:
Required to support `allow-past-years`, a reasonable bump
https://github.com/Lucas-C/pre-commit-hooks/releases
https://github.com/pre-commit/pre-commit/releases/tag/v3.2.0